### DISCUSSION TITLE
Truffle - Telegram API

### DISCUSSION THREAD

**Discussion by Khidirov** [ConsenSys-archive/truffle#5781] [General Q&A] [answered]
Hello!

I need a help with connecting [Truffle - Ganache - Metamask] where instead of using any browser I'd use a telegram bot for testing simple p2p transactions. Is it possible to make such thing or anyone already had experience with that? Any information will be helpful.

P.S. I'm newbie in solidity.
Thanks!

**Comment by apdan510** [comment]
Hey @Khidirov just to clarify, are you seeking the ability to connect Ganache to Metamask but instead of a browser it is a version of Metamask in Telegram?

Kind regards,
Alex

**Reply by Khidirov** [reply]
I assume that it will be possible to connect Ganache and Telegram only with the help of a MetaMask, but I could be wrong.  
The goal is to connect Truffle and Ganache to the Telegram bot and make transactions through the Telegram.

**Comment by Khidirov** [comment, marked as answer]
Found already, thanks for attention

**Reply by gnidan** [reply]
Glad you figured this out! How'd you do it?
